sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario2/UserChangeMailScenario2IntTest.java

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.integrationtest.scenario2;
+
+import static com.mercedesbenz.sechub.integrationtest.api.AssertMail.assertMailExists;
+import static com.mercedesbenz.sechub.integrationtest.api.TestAPI.*;
+import static com.mercedesbenz.sechub.integrationtest.api.TestAPI.getLinktToVerifyEmailAddressAfterChangeRequest;
+import static com.mercedesbenz.sechub.integrationtest.scenario2.Scenario2.USER_1;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.net.URI;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.mercedesbenz.sechub.integrationtest.api.IntegrationTestSetup;
+import com.mercedesbenz.sechub.integrationtest.api.TestUserDetailInformation;
+import com.mercedesbenz.sechub.integrationtest.api.TextSearchMode;
+
+public class UserChangeMailScenario2IntTest {
```

Comment:
Thank you. Integration test is really necessary for the changes and having an own dedicated one is a good idea here.